src/pcapexport.cpp

```diff
@@ -0,0 +1,63 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <assert.h>
+#include "pcapexport.h"
+
+/*
+	Useful documentation:
+	- https://wiki.wireshark.org/Development/LibpcapFileFormat
+	- http://www.tcpdump.org/linktypes.html
+*/
+
+#define DLT_USB_2_0 288
+
+PCapExport::PCapExport(const char* filename, USBModel& _model) : model(_model)
+{
+	fh = fopen(filename, "wb");
+	assert(fh);
+}
+
+PCapExport::~PCapExport(void)
+{
+	fclose(fh);
+}
+
+void PCapExport::write(void)
+{
+	writeGlobalHeader();
+
+	auto& aggregator = model.getAggregator();
+	for (int i = 0; i < aggregator.count(); i++) {
+		writePacket(*aggregator.value(i));
+	}
+}
+
+void PCapExport::writeGlobalHeader(void)
+{
+	pcap_hdr_t header;
+
+    header.magic_number = 0xA1B2C3D4;
```

Comment:
Why not use nanosecond resolution maigc (0xA1B23C4D)?

Reply:
Wasn't aware of it, now using it.

Reply:
Well, in the latest commit there is invalid magic. You use there 0xA1B2C34D while it should be 0xA1B23C4D.

Reply:
Fixed!